Dockerfile.test.db

```diff
@@ -7,26 +7,31 @@ COPY SQL/0000-00-01-Permission.sql /0000-00-01-Permission.sql
 COPY SQL/0000-00-02-Menus.sql /0000-00-02-Menus.sql
 COPY SQL/0000-00-03-ConfigTables.sql /0000-00-03-ConfigTables.sql
 COPY SQL/0000-00-04-Help.sql /0000-00-04-Help.sql
-COPY test/test_instrument/testtest.sql /test_instrument.sql
-COPY test/RBdata.sql /RBdata.sql
-COPY test/test_instrument/medical_history.sql /medical_history.sql
-COPY docs/instruments/radiology_review.sql /radiology_review.sql
+COPY docs/instruments/instrument_sql/aosi.sql  /aosi.sql
+COPY docs/instruments/instrument_sql/bmi.sql  /bmi.sql
+COPY docs/instruments/instrument_sql/medical_history.sql  /medical_history.sql
+COPY docs/instruments/instrument_sql/mri_parameter_form.sql  /mri_parameter_form.sql
+COPY docs/instruments/instrument_sql/radiology_review.sql  /radiology_review.sql
+COPY test/RBdata_consent_info_history.sql  /RBdata_consent_info_history.sql
+COPY test/RBfiles/*.sql /RBfiles/
+COPY test/RBdata_meta_prerequesites.sql /RBdata_meta_prerequesites.sql
 
 RUN echo "Use LorisTest;" | cat - \
     0000-00-00-schema.sql \
     0000-00-01-Permission.sql \
     0000-00-02-Menus.sql \
     0000-00-03-ConfigTables.sql \
     0000-00-04-Help.sql \
-    test_instrument.sql \
-    RBdata.sql \
+    aosi.sql \
+    bmi.sql \
     medical_history.sql \
-    radiology_review.sql > /docker-entrypoint-initdb.d/0000-compiled.sql 
+    mri_parameter_form.sql \
+    radiology_review.sql \
+    RBdata_meta_prerequesites.sql \
```

Comment:
RBdata_consent_info_history.sql  /

Reply:
Consent info history was removed in 20.0.0

I will update the docker